cbor/src/test/java/com/fasterxml/jackson/dataformat/cbor/mapper/BiggerDataTest.java

```diff
@@ -137,4 +140,33 @@ public void testRoundTrip() throws Exception
 		assertEquals(citm.topicSubTopics.size(), citm2.topicSubTopics.size());
 		assertEquals(citm.venueNames.size(), citm2.venueNames.size());
 	}
+
+	public void testRoundTripStringref() throws Exception
+	{
+		Citm citm0 = MAPPER.readValue(getClass().getResourceAsStream("/data/citm_catalog.json"),
+				Citm.class);
+		ObjectMapper mapper = new CBORMapper(
+		    new CBORFactory().enable(CBORGenerator.Feature.STRINGREF));
```

Comment:
Same here, builder over factory configuration (I know, many existing tests use this, trying to move over with new code).